docs/app/Examples/addons/Radio/Types/index.js

```diff
@@ -23,7 +25,14 @@ const RadioTypesExamples = () => (
       title='Radio Group'
       description='Radios can be part of a group.'
       examplePath='addons/Radio/Types/RadioGroup'
-    />
+    >
+      <Message warning>
+        Radios in a group must be
```

Comment:
I'm not aware of the differences between those, though the spaced string did give issues elsewhere.

Since space is removed between components that are separated by a new line, and the component's content should always be one space away from adjacent text, I've opted to include `&nbsp;` inside of components.  This way, no matter where the components are moved the spacing is correct without having to add/remove spacing elements every time you move the component.